api/internal/git/repospec_test.go

```diff
@@ -285,11 +287,104 @@ func TestNewRepoSpecFromUrl_Smoke(t *testing.T) {
 				GitSuffix: ".git",
 			},
 		},
+		{
+			name:      "t15",
+			input:     "https://github.com/kubernetes-sigs/kustomize//examples/multibases/dev/?submodules=0&ref=v1.0.6&timeout=300",
+			cloneSpec: "https://github.com/kubernetes-sigs/kustomize.git",
+			absPath:   notCloned.Join("/examples/multibases/dev"),
+			repoSpec: RepoSpec{
+				Host:      "https://github.com/",
```

Comment:
I remember you mentioning in another PR that we should probably rename `RepoSpec.Host` to `RepoSpec.SchemeHost`. I still think that's a good suggestion, and we can include it in this PR if you'd like.

Reply:
I actually thought that it was a behavior change the previous PR or recent PR added. I was wrong. This seems like the `Host` hasn't meant that for a long time. So I'm actually more comfortable now with doing that in a separate PR haha. Thanks for calling it out and being so open about it though!